README.md

```diff
@@ -1,8 +1,21 @@
 # ExceptionHunter
```

Comment:
Maybe we should include some points related to the configuration?
- Like default configuration runs in production/staging mode, if you want to run in dev you should change in the initializer file. 
- In order to disable the authentication you should remove the line where the dashboard user is declared. You can change the name of the dashboard user with the with the parameter -name during the install, in case you have another admin already going. 
- Exception Hunter will try to track the user responsible for the raised exception, so if this is wanted, one must declare the way of referring to the sessions of the application user. Most recurrent case would be `current_user` but you might have named it differently (e.g. `current_member`, `current_person`).